CMakeLists.txt

```diff
@@ -29,11 +31,20 @@ endif()
 cmake_minimum_required(VERSION 3.15)
 project(RealmJS)
 
-# Use ccache if available
-find_program(CCACHE_PROGRAM ccache)
-if(CCACHE_PROGRAM)
-    set_property(GLOBAL PROPERTY RULE_LAUNCH_COMPILE "${CCACHE_PROGRAM}")
-endif()
```

Comment:
Why would this not break the use of ccache?

Reply:
It looks like a mistake.